src/Components/Patient/PatientInfoCard.tsx

```diff
@@ -401,7 +401,9 @@ export default function PatientInfoCard(props: {
                       <span className="flex">
                         {consultation?.encounter_date && (
                           <div>
-                            Admission on:{" "}
+                            {consultation.suggestion === "DC"
+                              ? "Commenced on: "
+                              : "Admitted on: "}{" "}
```

Comment:
```suggestion
                              : "Admitted on: "}
```

We already have a space after `on: ` so let's not have 2 spaces.